Docker/Dockerfile

```diff
@@ -76,8 +76,8 @@ RUN git clone -b master --depth 1 https://github.com/EOSIO/eos.git --recursive \
     && cd eos \
     && cmake -H. -B"/tmp/build" -GNinja -DCMAKE_BUILD_TYPE=Release -DWASM_LLVM_CONFIG=/opt/wasm/bin/llvm-config -DCMAKE_CXX_COMPILER=clang++ \
        -DCMAKE_C_COMPILER=clang -DCMAKE_INSTALL_PREFIX=/opt/eos  -DSecp256k1_ROOT_DIR=/usr/local \
-    && cmake --build /tmp/build --target install
-
+    && cmake --build /tmp/build --target install \
+    && cd /tmp/build && cmake -DCMAKE_INSTALL_PREFIX=/opt/eos -P cmake_install.cmake
```

Comment:
I ran this and it didn't work kept getting ``` 'dir not found' ```
 I changed it to 
`cd /tmp/build && cmake -DCMAKE_INSTALL_PREFIX=/opt/eosio -P cmake_install.cmake`

installed.

Reply:
@mhollis1980 Well, yes, they changed eos to eosio recently... https://github.com/EOSIO/eos/commit/6af3424cbac711687db4fa4785d33d64eb8c4f3e#diff-f420623fda466cf1b6e26a839ad17fcd

Reply:
I'm wondering then should we update this PR to reflect those changes? What do you think?